readme.md

```diff
@@ -15,16 +15,24 @@ The clipboard content can also be cleared an shown in browse mode.
 
 Note: The above commands can be changed from NVDA menu, Preferences submenu, Input gestures dialog, Text review category.
 
-## Preferences Menu ##
-*	Clip Contents Designer settings: Allows to set a separator which can be used to find the text segments once the entire added text is pasted.
-It's also possible to choose if the added text will be appended or prepended, if available actions (add, clear clipboard, emulate copy and emulate cut) should be performed inmediately or after confirmation, and if confirmations will be requested always, just if text is contained in the clipboard, or if clipboard is not empty.
-Furthermore, it's possible to change the format and maximum number of characters of the clipboard text which will be shown in browse mode. Please, be aware that increasing this limit may produce issues if the clipboard contains large strings of text. The default limit is 100000 characters.
+## Clip Contents Designer settings ##
+
+This panel is available from NVDA's menu, Preferences submenu, Settings dialog.
+
+It contains the following controls:
+
+* Type the string to be used as a separator between contents added to the clipboard: Allows to set a separator which can be used to find the text segments once the entire added text is pasted.
+* Add text before clip data: It's also possible to choose if the added text will be appended or prepended.
+* Select the actions which require previous confirmation: You can choose, for each action available, if it should be performed inmediately or after confirmation. Available actions are: add text, clear clipboard, emulate copy and emulate cut.
+* Request confirmation before performing the selected actions when: You can select if confirmations will be requested always, just if text is contained in the clipboard, or if clipboard is not empty (for example if you've copied a file, not text).
+* Format to show the clipboard text in browse mode: If you're learning HTML markup language, you may choose Preformatted text in HTML or HTML as shown in a web browser, to have an idea of how your HTML code will be rendered by NVDA in a browser. For example, write some HTML tags like h1, h2, li, pre, etc., select and copy the text to clipboard, and use clipContentsDesigner add-on to show the text in a browseable message. You can change the format to raw text, and then the tags won't be translated in the browser, just will be shown as plain text.
```

Comment:
I still do not understand the difference between the two HTML options. Maybe a sublist with a more detailed description of each option would be clearer? Sorry if I missed something obvious.

Additional note: Also maybe it is a bit off-topic for this PR. But I would have expected the raw text to be the first option, in the doc but also in the UI, since it seems to me the most common use case, i.e. someone wanting to review the content of the clipboard without any interpretation. And also raw text to be the default option (if not already).

Reply:
I'll improve the description to explain the difference between preformatted and HTML as shown in a browser. The difference is that preformatted HTML shows spaces and line breaks, while HTML as shown in a browser compacts several spaces in a unique blank character. This is explained in [HTML pre tag - W3Schoolswww.w3schools.com › tags › tag_pre ](https://www.w3schools.com/tags/tag_pre.asp).
Imo, the most useful option for this add-on is the preformatted text, since, if we want to show the raw text, we can more easily paste the text in an editor like Notepad, but we can use the capabilities of NVDA with this add-on to show how the HTML code is rendered, what may not be very easy to see in other ways.

Reply:
> Imo, the most useful option for this add-on is the preformatted text, since, if we want to show the raw text, we can more easily paste the text in an editor like Notepad, but we can use the capabilities of NVDA with this add-on to show how the HTML code is rendered, what may not be very easy to see in other ways.

I have a complete opposite reasoning from you:
* the raw text seems to me the most useful option for a wider range of users because only few users play with HTML code. The use case with raw text is to copy 3 pieces of text from 3 various places and to be able to paste all together at the same place. That's the reason why I have downloaded this add-on. Of course, I may use notepad as I did before, but this add-on allows me to spare a lot of keystrokes to perform frequently such multiple-copy/paste actions.
* On the contrary, when I code some HTML, I have the source code in the editor and the result opened in my browser. If I make some slight change in the source code, I just have to Alt+Tab to my browser and press F5 to see the result.

Now that I have explained with more details my use cases, I'll let you the final decision. You are the add-on author after all. And this is just a matter of default option and order in the documentation. That's not so much!